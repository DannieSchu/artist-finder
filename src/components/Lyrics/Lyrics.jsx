import React from 'react';
import PropTypes from 'prop-types';
import styles from '../detail.css';

const Lyrics = ({ lyrics = 'No lyrics available' }) => (
  <section className={styles.Detail}>
    <h3>Lyrics</h3>
    <p>{lyrics}</p>
  </section>
);

Lyrics.propTypes = {
  lyrics: PropTypes.string
};

export default Lyrics;

