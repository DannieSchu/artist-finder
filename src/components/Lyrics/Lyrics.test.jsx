import React from 'react';
import { shallow } from 'enzyme';
import Lyrics from './Lyrics.jsx';

describe('Lyrics', () => {
  it('matches a snapshot', () => {
    const lyrics = 'Good time for a change See, the luck I\'ve had Can make a good man Turn bad So please please please Let me, let me, let me Let me get what I want This time';
    
    const wrapper = shallow(
      <Lyrics
        lyrics={lyrics}
        artist="The Smiths"
        recording="Please, Please, Please Let Me...Go To Bed" />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
