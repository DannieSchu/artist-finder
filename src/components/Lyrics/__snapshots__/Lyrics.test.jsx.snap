// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lyrics matches a snapshot 1`] = `
<section
  className="Detail"
>
  <h3>
    Lyrics
  </h3>
  <p>
    Good time for a change See, the luck I've had Can make a good man Turn bad So please please please Let me, let me, let me Let me get what I want This time
  </p>
</section>
`;
